### PR TITLE
Update classification model FLOPs with `imgsz=224`

### PR DESCRIPTION
**Pull request by Y-T-G**
The higher FLOPs count makes the models seem slower, although they were calculated at `imgsz=640` which many may overlook.

Change the FLOPs to be the one calculated at `imgsz=224` since that is the default imgsz for the pretrained YOLO classification models, and it is also what's used by most other classification models and shown in FLOPs comparison, such as the one in TorchVision: https://pytorch.org/vision/main/models.html#table-of-all-available-classification-weights

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary  
Updated classification model configurations and documentation to improve accuracy and consistency, particularly for FLOPs calculations and class counts.

### 📊 Key Changes  
- **FLOPs Adjustments**: Corrected FLOPs (Floating Point Operations) values for classification models in documentation and configuration files, reducing them significantly for better accuracy.  
- **Class Count Update**: Increased the number of classes from 10/80 to 1000 in model configuration files to align with ImageNet dataset standards.  
- **Documentation Updates**: Updated tables in README files (English and Chinese) and other docs to reflect corrected FLOPs values and consistent formatting.  

### 🎯 Purpose & Impact  
- **Improved Accuracy**: Ensures FLOPs values are correctly calculated, providing users with more reliable performance metrics.  
- **Enhanced Usability**: Aligning class counts with ImageNet improves compatibility and usability for classification tasks.  
- **Clearer Documentation**: Updated documentation ensures users have accurate and consistent information, reducing confusion.  

🚀 These changes enhance the clarity, reliability, and usability of Ultralytics' classification models for developers and researchers alike!

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Y-T-G, thank you for submitting an `ultralytics/ultralytics` 🚀 PR! To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA/) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_  — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing/). Don’t hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

This is an automated response to help guide your contribution. An Ultralytics engineer will review your PR and provide feedback soon. Stay tuned! 🤖

**Comment by glenn-jocher** [comment]
@Y-T-G this looks good. How did you do the calculations? 

We still have a long-standing issue of showing FLOPs at 640 instead of `imgsz`. I've looked at it a few times but it's not a simple fix unfortunately.

**Comment by Y-T-G** [comment]
@glenn-jocher  I used the `imszg` argument if `model.info()`

```python
from ultralytics import YOLO
[YOLO(f"yolo11{s}-cls.yaml").model.info(imgsz=224) for s in "nsmlx"]
```

**Comment by glenn-jocher** [comment]
@Y-T-G awesome PR!

**Comment by UltralyticsAssistant** [comment]
🌟 **Merged and Magnificent!** 🌟  

Thank you so much, @Y-T-G and @glenn-jocher, for your stellar contributions to this PR! Your meticulous updates to the classification model configurations and documentation represent the kind of attention to detail that drives innovation and excellence.  

As Benjamin Franklin once said, *"Without continual growth and progress, such words as improvement, achievement, and success have no meaning."* This PR embodies that spirit—improving accuracy, enhancing usability, and making our tools even more reliable for the community.  

Your work not only sharpens the technical edge of Ultralytics but also makes life easier for developers and researchers around the globe. 🚀 We deeply appreciate your dedication and teamwork—thank you for helping us reach new heights! 🙌
